include/abb_librws/v2_0/rw/elog.h

```diff
@@ -32,17 +32,18 @@ namespace abb ::rws ::v2_0 ::rw ::elog
      * @param lang The language in which domain names will be returned. English default.
      * @return std::vector<std::pair<int, std::string>> - List of domain id and domain name in provided language
      */
-    std::vector<std::pair<int, std::string>> getElogDomains(RWSClient &client, std::string lang = Language::ENGLISH);
+    std::vector<std::pair<int, std::string>> getElogDomains(RWSClient& client, std::string lang = Language::ENGLISH);
 
     /**
      * @brief Get a specific elog message in the given domain.
      * https://developercenter.robotstudio.com/api/rwsApi/elog_domain_get_page.html
      *
      * @param client RWSClient
      * @param domain domain number
-     * @param seqnum sequence number
+     * @param seqnum sequence number to start from. If -1, the last @param limit messages are returned. Default -1.
      * @param lang The language in which messages will be returned. English default.
+     * @param limit The number of messages to return. Default 50.
      * @return ElogMessage structure
      */
-    ElogMessage getElogMessage(RWSClient &client, int const domain, int const seqnum, std::string const &lang = Language::ENGLISH);
+    std::vector<ElogMessage> getElogMessages(RWSClient& client, int const domain, int const seqnum = -1, std::string const& lang = Language::ENGLISH, int const limit = 50);
```

Comment:
[nitpick] We don't know what exceptions this method can throw from doc, hence handling them can be troublesome

Reply:
I added error handling for all of the mentioned cases, we should not throw any exceptions in this case.